flow/record/adapter/sqlite.py

```diff
@@ -141,6 +141,7 @@ def read_table(self, table_name: str) -> Iterator[Record]:
         # flow.record is quite strict with what is allowed in fieldnames or decriptor name.
         # While SQLite is less strict, we need to sanitize the names to make them compatible.
         table_name_org = table_name
+        table_name_org = table_name_org.replace('"', '""')
```

Comment:
```suggestion
        table_name_org = table_name.replace('"', '""')
```
Also what effect does this have exactly? An insert statement with a table name of `a"b` will look like `INSERT INTO "a""b"`? How does that work? Is this some escaping behaviour of SQLite/DuckDB?

Reply:
This is indeed escaping behaviour that works both on SQLite and DuckDB:

```shell
> duckdb
v0.8.1 6536a77232
Enter ".help" for usage hints.
Connected to a transient in-memory database.
Use ".open FILENAME" to reopen on a persistent database.
D create table 'a"b' (foo text);
D .schema
CREATE TABLE IF NOT EXISTS "a""b"(foo VARCHAR);;
D insert into "a""b" values ('bar');
D select * from "a""b";
┌─────────┐
│   foo   │
│ varchar │
├─────────┤
│ bar     │
└─────────┘
D
```

```shell
> sqlite3
SQLite version 3.28.0 2019-04-15 14:49:49
Enter ".help" for usage hints.
Connected to a transient in-memory database.
Use ".open FILENAME" to reopen on a persistent database.
sqlite> create table 'a"b' (foo text);
sqlite> .schema
CREATE TABLE IF NOT EXISTS 'a"b' (foo text);
sqlite> insert into "a""b" values ('bar');
sqlite> select * from "a""b";
bar
sqlite>
```